sqlmesh/dbt/basemodel.py

```diff
@@ -190,13 +192,18 @@ def relation_info(self) -> AttributeDict[str, t.Any]:
             }
         )
 
+    @property
+    def config_info(self) -> AttributeDict[str, t.Any]:
```

Comment:
`config_info`? Should this just be `attribute_dict()` similarly to `dict()`?

Reply:
Wil rename